packages/xdl/src/EmbeddedAssets.ts

```diff
@@ -46,57 +48,100 @@ async function _maybeWriteArtifactsToDiskAsync(config: EmbeddedAssetsConfigurati
     androidSourceMap,
   } = config;
 
+  let androidBundlePath;
+  let androidManifestPath;
+  let androidSourceMapPath;
+  let iosBundlePath;
+  let iosManifestPath;
+  let iosSourceMapPath;
+
+  // set defaults
+  if (pkg.dependencies['expo-updates']) {
+    const context = StandaloneContext.createUserContext(projectRoot, exp);
+    const { supportingDirectory } = IosWorkspace.getPaths(context);
+    const androidAssetsDir = path.join(projectRoot, 'android', 'app', 'src', 'main', 'assets');
+
+    await fs.ensureDir(supportingDirectory);
+    await fs.ensureDir(androidAssetsDir);
+
+    androidBundlePath = path.join(androidAssetsDir, 'shell-app.bundle');
```

Comment:
would be nice to change these to names that make more sense for non-shell-app projects